ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -84,10 +91,29 @@ export class ImportCommand extends IronfishCommand {
 
     process.stdin.off('data', onData)
 
-    return JSONUtils.parse<AccountImport>(data)
+    try {
+      const decodedWords = bech32m.bech32m.decode(data).words
+      const decodedBytes = bech32m.bech32m.fromWords(decodedWords)
+      return JSONUtils.parse<AccountImport>(decodedBytes.toString())
+    } catch (e) {
+      return JSONUtils.parse<AccountImport>(data)
+    }
```

Comment:
Same as above, also if this is a reusable pattern, should move it into its own method to avoid duplication.